### DISCUSSION TITLE
Necesito ayuda.

### DISCUSSION THREAD

**Discussion by Kira1010geek01** [wazuh/wazuh#23933] [General]
Tengo una máquina virtual con Ubuntu 22.04
Y quiero instalar wazuh pero tengo el siguiente inconveniente:
El primer paso para instalar wazuh sería ejecutar el siguiente código (obviamente como root sudo su):
curl -s0 https://packages.wazuh.com/4.7/wazuh-install.sh

Ese comando me deja ejecutarlo sin problemas.
Pero el siguiente comando no me deja:

sudo bash ./wazuh-install.sh -a

Al ejecutar ese script me sale el siguiente error:
No such file or directory
Cómo podría instalar wazuh y librarme de ese error?

![IMG_20240605_205932.jpg](https://github.com/wazuh/wazuh/assets/169662532/6d532665-de53-4cec-8dd4-e0c1d8fd342a)

**Comment by rauldpm** [comment]
Hi @Kira1010geek01 

English: Since the repositories are public and are widely consulted by non-Spanish-speaking users, I have to answer you in English so that everyone can take advantage of the discussion, in any case, I leave you the Spanish translation at the end of the comment.

Spanish: Dado que los repositorios son públicos y son ampliamente consultados por usuarios no hispanohablantes, tengo que responderte en inglés para que todos aprovechen la discusión, en cualquier caso les dejo la traducción al español al final del comentario.

---

The error you show is probably caused because the script has not been downloaded to the VM using `curl`.

From what I see, the `curl` command you shared is:

> curl -s`0` https://packages.wazuh.com/4.7/wazuh-install.sh

When it should be:

> curl -s`O` https://packages.wazuh.com/4.7/wazuh-install.sh

Note that it is not a `0` (zero) but a capital letter `O` (like Office)

Can you try with that modification again and tell us if you still have the problem?

---

<details><summary>Spanish translation</summary>


El error que muestras probablemente se deba a que el script no se descargó en la VM usando `curl`.

Por lo que veo, el comando `curl` que compartiste es:

> curl -s0 https://packages.wazuh.com/4.7/wazuh-install.sh

Cuando debería ser:

> curl -sO https://packages.wazuh.com/4.7/wazuh-install.sh

Ten en cuenta que no es un `0` (cero) sino una letra mayúscula `O` (como Oficina)

¿Puedes intentarlo de nuevo con esa modificación y decirnos si aún tienes el problema?


</details>

**Reply by Kira1010geek01** [reply]
Muchas gracias... Al no copiar y pegar sino que  hice el comando a mano tuve ese error de tipografía y ni cuenta me di, no era un 0 era una O mayúscula

**Reply by rauldpm** [reply]
Did you manage to download the script and run it?

**Reply by Kira1010geek01** [reply]
Si, muchas gracias.
